src/xunit.performance.api/Kernel32.cs

```diff
@@ -87,5 +87,22 @@ private enum ConditionMask : byte
             ulong dwlConditionMask,
             TypeMask dwTypeBitMask,
             ConditionMask dwConditionMask);
+
+        [Flags]
+        public enum CtrlTypes : uint
+        {
+            CTRL_C_EVENT = 0,
+            CTRL_BREAK_EVENT = 1,
+            CTRL_CLOSE_EVENT = 2,
+            CTRL_LOGOFF_EVENT = 5,
+            CTRL_SHUTDOWN_EVENT = 6
+        }
+
+        public delegate bool PHANDLER_ROUTINE(CtrlTypes dwCtrlType);
+
+        [DllImport("kernel32.dll", SetLastError = true, CallingConvention = CallingConvention.Winapi)]
```

Comment:
Why are we using P/Invokes rather than the CoreFx support that wraps this, e.g. Console.CancelKeyPress?  Is there something we're missing in the Console implementation (which works cross-platform) that this needs?